01-davinci.md

```diff
@@ -0,0 +1,119 @@
+These are implemented as *Tools*, which are shared between Algorithms.
+
+**Options**
+To make all of this configurable, Gaudi allows you to set properties of *Algorithms* and *Tools* from a Python script, called an *option* file.
+In an option file, you can specify which Algorithms are run in which order, and set their properties (strings, integers, doubles, and lists and dicts of these things can be set).
+You can then start the Gaudi EventLoop using this option file, and it will set up and run the corresponding C++ objects with specified settings.
+
+You can find comprehensive documentation in the [Gaudi Doxygen](https://proj-gaudi.web.cern.ch/proj-gaudi/releases/latest/doxygen/) or the [Gaudi Manual](http://lhcb-comp.web.cern.ch/lhcb-comp/Frameworks/Gaudi/Gaudi_v9/GUG/GUG.pdf).
+
+Usually, you will work with one of the LHCb software projects that are based on Gaudi.
+One of the most important ones is *DaVinci*, which provides lots of *Algorithms* and *Tools* for physics analysis.
+
+You can run DaVinci using the following command:
+```bash
+lb-run DaVinci v36r6 gaudirun.py
+```
+
+This will run the `gaudirun.py` command using version v36r6 of DaVinci.
+`gaudirun.py` is a script that sets up the EventLoop.
+You should get the following output:
+
+```
+# setting LC_ALL to "C"
+ApplicationMgr    SUCCESS 
+====================================================================================================================================
+                                                   Welcome to DaVinci version v36r6
+                                          running on lxplus0077.cern.ch on Thu May 28 11:09:31 2015
+====================================================================================================================================
+ApplicationMgr       INFO Application Manager Configured successfully
+HistogramPersis...WARNING Histograms saving not required.
+ApplicationMgr       INFO Application Manager Initialized successfully
+ApplicationMgr       INFO Application Manager Started successfully
+EventSelector        INFO End of event input reached.
+EventLoopMgr         INFO No more events in event selection 
+ApplicationMgr       INFO Application Manager Stopped successfully
+EventLoopMgr         INFO Histograms converted successfully according to request.
+ToolSvc              INFO Removing all tools created by ToolSvc
+ApplicationMgr       INFO Application Manager Finalized successfully
+ApplicationMgr       INFO Application Manager Terminated successfully
+```
+
+During this run, DaVinci didn't do anything: We didn't specify any algorithms to run or any data to run over.
+Usually, you will write an option file (e.g. `options.py`) and specify it as an argument to `gaudirun.py`:
+
+```bash
+lb-run DaVinci v36r6 gaudirun.py options.py
+```
+
+An `option.py` is just a regular Python script that specifies how to set things up in the software.
```

Comment:
Maybe add you can specify several options files and they will be used in sequence to configure Gaudi. This is very useful when you build resuable components which just have small differences (data/MC, for example) that can be extracted in separate options files.